src/containers/Auth/ErrorPage.js

```diff
@@ -57,40 +64,37 @@ const ErrorPage = (props) => {
         <PageLayout alignItems="left">
           <h1
             className="p-0"
-            style={{ marginBottom: "50px", color: "#ff9800", fontSize: "55px" }}
+            style={{ marginBottom: "10px", color: "#ff9800", fontSize: "55px" }}
           >
-            Oops
+            Oops!
           </h1>
           <p style={style.helpText} className="pt-0">
-            An error occured, please contact your administrator to get it fixed
-            ...!!!{" "}
+            An error occured, please contact your administrator to get it fixed.
           </p>
           {errorInfo && errorInfo !== null ? (
             <>
               <Paper variant="outlined" style={{ marginBottom: "50px" }}>
-                <span style={{ color: "dodgerblue", fontSize: "16px" }}>
-                  Code : {errorInfo.code}
-                </span>
-                <br></br>
-                <span style={{ color: "dodgerblue", fontSize: "16px" }}>
-                  Message : {errorInfo.message}
-                </span>
+                {errorInfo.code != null ? (
+                  <p style={{ color: "dodgerblue", fontSize: "16px" }}>
+                    Code : {errorInfo.code}
+                  </p>
+                ) : null}
+                {errorInfo.message != null ? (
+                  <p style={{ color: "dodgerblue", fontSize: "16px" }}>
+                    Message : {errorInfo.message}
+                  </p>
+                ) : null}
+                {errorInfo.reason != null ? (
+                  <p style={{ color: "dodgerblue", fontSize: "16px" }}>
+                    Reason : {errorInfo.reason}
+                  </p>
+                ) : null}
               </Paper>
             </>
           ) : null}
           <Button variant="contained" color="primary" onClick={moveToDashboard}>
             Go to Homepage
           </Button>
-          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
why did we remove this ? can we still have this and point to troubleshooting. We can improve troubleshooting docs with time and may be it will be helpful for folks.

Reply:
My bad, I rechecked the troubleshoot page, there is no section explaining errors in login flows there. So it's dead-end for the end-user.
I think we should first add some potential login errors in troubleshooting then point user to troubleshoot page. What do you think?
Few example in ory's document: https://www.ory.sh/docs/troubleshooting/oidc-redirect-url-cname-OTA-092022-01

Reply:
thanks @akshay196 for the detailed explaination !